internal/internal_test.go

```diff
@@ -992,6 +993,52 @@ func TestConfigInvalid(t *testing.T) {
 	}
 }
 
+func TestConfigWithValidProtoDescriptor(t *testing.T) {
+	m, err := plugins.New([]byte{}, "test", inmem.New())
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	in := `{"proto-descriptor": "../test/files/combined.pb"}`
+	_, err = Validate(m, []byte(in))
+	if err != nil {
+		t.Fatal(err)
+	}
+}
+
+func TestConfigWithInvalidProtoDescriptor(t *testing.T) {
+	for name, path := range map[string]string{ // desc -> path
+		"nonexistent":     "this/does/not/exist",
+		"other file type": "../test/files/book/Book.proto",
+	} {
+		t.Run(name, func(t *testing.T) {
+			m, err := plugins.New([]byte{}, "test", inmem.New())
+			if err != nil {
+				t.Fatal(err)
+			}
+
+			in := fmt.Sprintf(`{"proto-descriptor": "%s"}`, path)
+			_, err = Validate(m, []byte(in))
+			if err == nil {
+				t.Error("expected err, got nil")
+			}
+		})
+	}
+}
+
+func TestConfigWithNonexistentProtoDescriptor(t *testing.T) {
+	m, err := plugins.New([]byte{}, "test", inmem.New())
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	in := `{"proto-descriptor": "this/does/not/exist"}`
+	_, err = Validate(m, []byte(in))
+	if err == nil {
+		t.Error("expected err, got nil")
+	}
+}
+
```

Comment:
`TestConfigWithNonexistentProtoDescriptor ` got covered in `TestConfigWithInvalidProtoDescriptor `.

Nit: We could combine `TestConfigWithValidProtoDescriptor ` and `TestConfigWithInvalidProtoDescriptor` into a single table driven test.